gdal/swig/python/scripts/gdal2tiles.py

```diff
@@ -489,7 +493,7 @@ class GlobalGeodetic(object):
        WMS, KML    Web Clients, Google Earth  TileMapService
     """
 
-    def __init__(self, tmscompatible, tile_size=256):
+    def __init__(self, tmscompatible: Optional[Any], tile_size: int = 256) -> None:
```

Comment:
Yes, that would indeed make more sense IMO. Currently the possible values are True and None but supporting False would make sense. I didn't do it first as it would be a non-typing change